### PR TITLE
Update README file

### LINKED ISSUE #74: Update project details documentation page

### PR DESCRIPTION
**Pull request by annagabriel-hash**
### Tasks
- [x] Update project description details
- [x] Update prerequisites and installtion
- [x] Update installation guide
- [x] Update usage
- [x] Update contribution

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,58 +1,119 @@
...
-  $ bundle install
+# Drop that beat app   ![build](https://github.com/lea927/drop-that-beat/actions/workflows/rspec.yml/badge.svg)
+
+
+![Drop that beat app](https://user-images.githubusercontent.com/79251819/136054424-c82d19df-477b-4dbc-8f23-0185d0658d44.png)
+The developers created this app to bond with their love of music. We hope that users will enjoy listening to their favorite music and learn other songs they can tune into 😃 🎵🎵🎵
+
+## Features 🎵🎵🎵
+
+- Create and edit an account
```
Comment (lea927) [inline README.md:9]: You can change this to something like Register and update account details.

File: README.md [hunk 2]
```diff
@@ -1,58 +1,119 @@
...
+- Create, edit and delete a game rooms. Game rooms created are accessible to other registered users.
```
Comment (lea927) [inline README.md:10]: Users can create, edit and delete game rooms. The game rooms created are accessible to other registered users.

File: README.md [hunk 3]
```diff
@@ -1,58 +1,119 @@
...
+- Search tracks and add those tracks in the game room. 🎵
```
Comment (lea927) [inline README.md:11]: Search tracks which connected to Apple's iTunes Search API and add those tracks in the game room.

File: README.md [hunk 4]
```diff
@@ -1,58 +1,119 @@
...
-```
-  $ git clone git@github.com:<username>/rails-base-project.git <name of your app>
-  $ cd <name of your app>
-```
-
-#### Setup
-
-```
-  $ bundle install
+# Drop that beat app   ![build](https://github.com/lea927/drop-that-beat/actions/workflows/rspec.yml/badge.svg)
```
[the remark this answers is not in the document]
Comment (annagabriel-hash) [reply README.md]: Thanks for the suggestion! 
Please see commit 781c05b

File: README.md (hunk 3 above)
Comment (annagabriel-hash) [reply README.md]: Please see commit 7cf06e2 to resolve this 😃

**Pull request merged**